build/instrumentation.build.js

```diff
@@ -74,7 +71,7 @@ export function createConfig({
       },
       ...output
     },
-    preserveEntrySignatures: 'strict',
+    preserveEntrySignatures: false,
```

Comment:
We've changed this property value to `false` so `rollup` does not create a `facade chunk` for our entry point, which was only doing a `dynamic import`.

For further information, you can check [rollup's docs](https://rollupjs.org/guide/en/#preserveentrysignatures).

Reply:
Nice change, now I think we have a great optimised way to progressively  of loading the application